### ISSUE TITLE
Help document

### ISSUE THREAD

**Issue by T8z5h3** [microsoft/winfile#64]
hi,
can we have the help document format updated to support the newer o.s's?

**Comment by Anixx** [comment]
Install this program from Microsoft:
https://www.microsoft.com/en-us/download/details.aspx?id=35449

and the .hlp files will work well.
![clipboard_20180410](https://user-images.githubusercontent.com/1271736/38570156-b4f2f0dc-3cdc-11e8-9261-88a115e748e6.png)

**Comment by T8z5h3** [comment]
thanks

**Comment by NazmusLabs** [comment]
yeah, it is not supported on Windows 10. The help viewer was deprecated in Windows Vista. And while I expected it to be dropped by Windows 7, Microsoft has gone out of their way of supporting it up to Windows 8.1. But the writing was on the wall for long enough. There's now no way to actually open and use .hlp files on Windows 10.

HTML Help is fully supported and isn't even deprecated, yet. So, that's the simplest way forward.

But I would rather just have a documentations folder in GitHub and just dump the content of the HTML Help in GitHub, and redirect the help link in the menu to the browser. That's how most apps today work, and it allows the app to go the latest and most up to date documentation.

**Comment by Anixx** [comment]
In many settings people have no internet access or restricted access to the browser. Also, the browser is usually a very heavy-weight application that takes time to load and consumes a lot of memory which may be not justified for just looking for file manager help.

**Comment by NazmusLabs** [comment]
@Anixx I know. My suggestion was to have it as a temporary measure until a more compatible version is working. It's better than having nothing. Windows 10 is nearing half of all windows userbase, and growing, meaning the .hlp is useless. Perhaps, just even add an additional menu item to view the documentation on the web in addition to the existing help doc.

I am hoping that we can have the help file in an browser viewable HTML format that can open allow the help menu to open the help doc locally, but within the web browser.

A lot of apps do that. Why bother with writing help applications if you can just provide the html files locally and use the default browser to open it. The browser gives the best reading experience anyway.

**Comment by Anixx** [comment]
Help apps are far better the browser for reading help. They open faster, consume less resources, allow to better navigate the topics, keyword search, easier to create source files, etc. Try to create a web page with tree view. While it is possible, it is complicated and may bring a lot of bugs in each case while in help files you only bother with content, not with engine and scripting.

**Comment by NazmusLabs** [comment]
@Anixx I have looked through the html help files for the app. It doesn't even need a tree view because every page in the tree view has only a couple of sentences. The entire help documentation would only require one or two html web pages, all of which can be done with a simple hyperlink. Secondly, the HTML Help viewer is ancient, outdated, and doesn't not display correctly in my 150% display scaling and is blurry (I'm visually impaired, I need this scaling). As for keyword search, you can use ctrl+F and search for any keyword you want.

**Comment by tomrow** [comment]
@NazmusLabs I guess those HTML files eventually get compiled into a CHM file included with the download.

**Comment by justingoldberg** [comment]
> In many settings people have no internet access or restricted access to the browser. Also, the browser is usually a very heavy-weight application that takes time to load and consumes a lot of memory which may be not justified for just looking for file manager help.

True. This is well illustrated by the speed differences circa 1998 to ~2003 between loading a newer html help (.chm) and the older .hlp format. The Web/Active Desktop didn't add anything to the well-designed RTF formatted help.
